programs/nodeop/CMakeLists.txt

```diff
@@ -43,7 +43,6 @@ target_link_libraries( ${NODE_EXECUTABLE_NAME}
         PRIVATE sysio_chain_wrap fc ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS}
         Boost::dll )
 
-
```

Comment:
I know it is nit-picky, but if you add back in this line then there will be no changes for this file

Reply:
not going to try to change the entire commit merge history to avoid removing an unnecessary blank line

Reply:
(the line was removed in `83a453b4cd Merge branch 'leap3_with_history' into leap5_merged`)